src/rules/converters/file-name-casing.ts

```diff
@@ -0,0 +1,56 @@
+import { RuleConverter } from "../converter";
+
+const IGNORE_CASE_NOTICE = "ESLint (Unicorn plugin) does not support ignore as case";
```

Comment:
```suggestion
const IGNORE_CASE_NOTICE = "ESLint (Unicorn plugin) does not support the 'ignore' case.";
```

...to be a little more clear. Other `notices` end with periods too.